impl/src/rendering/templates/crate.BUILD.template

```diff
@@ -30,7 +30,7 @@ licenses([
 
 # Generated Targets
 {%- set crate_name_sanitized = crate.pkg_name | replace(from="-", to="_") -%}
-{%- if crate.build_script_target -%}
+{%- if crate.build_script_target %}
 {%      include "templates/partials/build_script.template" %}
```

Comment:
Nit: I think there should be an empty line added here to match style of other targets. There is an empty line following `# Generated Targets` in all other cases.

Reply:
Can you suggest a change for me to apply?